getting-started/truffle/yarn.lock

```diff
@@ -0,0 +1,10153 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
There seem to be two truffle files. Please delete the one outside the getting started folder.